### ISSUE TITLE
Delete download history

### ISSUE THREAD

**Issue by xcsh** [TeamNewPipe/NewPipe#961]
- [y] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [y] I checked if the issue/feature exists in the latest version.

Can you add a feature to clear the download history ? Right now deleting a download in newpipe completly removes the file.

Also when downloading audio i have a .giga file along with my audio file. What is its purpose ? Isn't it supposed to be removed once the download is finished ?

**Comment by theScrabi** [comment]
That yet is another issue that belongs to the "we have to remove gigaget downloader" thing :/

**Comment by xcsh** [comment]
Deleting the download history is also related to gigaget ?

**Comment by theScrabi** [comment]
Yes

**Comment by TobiGr** [comment]
in #1759
